test/honeybadger/json_test.exs

```diff
@@ -0,0 +1,48 @@
+defmodule Honeybadger.JSONTest do
+  use Honeybadger.Case
+
+  alias Honeybadger.{Notice, JSON}
+
+  describe "encode" do
+    test "encodes notice" do
+      notice = Notice.new(%RuntimeError{message: "oops"}, %{}, [])
+      assert JSON.encode(notice) == Jason.encode(notice)
+    end
+
+    defmodule Req do
```

Comment:
Can we move this outside of the `describe` call? It always trips me up when functions or modules are declared within a `describe` because it looks like they are scoped to the block but they are actually scoped to the module.